src/site/content/en/payments/payments.11tydata.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  // Tags are inherited by all posts.
+  tags: ['pathItem', 'payments'],
+  path: {
+    // Slug is used by landing pages like / and /learn to link to this path.
+    // Because it affects urls, the slug should never be translated.
+    slug: 'payments',
+    cover: '/images/collections/payments.svg',
+    title: 'Web Payments',
+    updated: 'May 25, 2020',
+    description: `Easy way to implement payments on the web.`,
+    overview: `In this section you'll learn how to use Web Payments APIs.`,
+    topics: [
+      {
+        title: 'Web Payments',
+        pathItems: ['setting-up-a-payment-method'],
```

Comment:
Is the plan to add more docs to this learning path before we land it? I'm not sure if we want to ship a learning path with only one item in it.

Reply:
There'll be 3 or 4 items here by Monday, another 3-4 coming in a couple of weeks later.

Reply:
ok sounds good :)